src/main/java/frc/team166/robot/subsystems/Drive.java

```diff
@@ -64,16 +66,17 @@
     });
 
     final static double AUTOMATIC_ROBOT_FORWARD_SPEED = .2;
-    final static double ABSOLUTE_TOLERANCE_ANGLE = 3;
+    final static double ABSOLUTE_TOLERANCE_ANGLE = 1;
```

Comment:
Are these things supposed to be preferences?

Reply:
I believe these are both used as the defaults for the preferences. This just ensures that the default is the same in all places we use the preference.